### ISSUE TITLE
subform.repeatable-table small screen issues

### ISSUE THREAD

**Issue by ciar4n** [joomla/joomla-cms#20210]
### Steps to reproduce the issue
Issue occurs when multiple fields are placed within a single cell for subform.repeatable-table. Add the following after https://github.com/joomla/joomla-cms/blob/staging/modules/mod_login/mod_login.xml#L26

```
<field name="sebform1" type="subform" label="subform"
	layout="joomla.form.field.subform.repeatable-table" multiple="true" groupByFieldset="true">
	<form>
		<fieldset name="group1" label="group 1">
			<field type="text" name="text" label="text 1"/>
			<field type="text" name="text2" label="text 2"/>
			<field type="text" name="text3" label="text 3"/>
		</fieldset>
		<fieldset name="group2" label="group 2">
			<field type="text" name="text4" label="text 4"/>
			<field type="textarea" name="textarea" label="textarea"/>
		</fieldset>
	</form>
</field>
```
Open login module settings. Check repeatable field styling on reduced browser width.

### Expected result
Something along the lines of...

![image](https://user-images.githubusercontent.com/2803503/39093686-631fd3b8-461b-11e8-9cb8-9be7b126b4f7.png)

### Actual result
A mess....

![image](https://user-images.githubusercontent.com/2803503/39093615-0df56854-461a-11e8-8139-1532842e74cd.png)

### Additional comments
I can fix once https://github.com/joomla/joomla-cms/pull/20209 is merged.

**Comment by brianteeman** [comment]
@ciar4n now that #20209 has been merged can you do the fix please

**Comment by ciar4n** [comment]
Sorry for delay. Missed your comment.

PR created.. #22335

**Comment by ghost** [comment]
closed as having Pull Request #22335
